lib/fauna/objects.rb

```diff
@@ -0,0 +1,111 @@
+module Fauna
+  ##
+  # A Ref.
+  #
+  # Reference: {FaunaDB Special Types}[https://faunadb.com/documentation#queries-values-special_types]
+  class Ref
+    # The raw ref string.
+    attr_accessor :ref
```

Comment:
I don't like having a method named `ref` returning something that's not a `Ref`. `str` sounds better to me.